src/containers/SearchPage/components/form/SearchContentForm.tsx

```diff
@@ -130,7 +129,7 @@ const SearchContentForm = ({ search: doSearch, searchObject: search, subjects, l
     {
       name: getTagName(search.subjects, subjects),
       type: 'subjects',
-      width: 25,
+      width: 50,
```

Comment:
Kanskje sette users til 50 også for å unngå at valgt filter havner på samme linje som søkefelta.